salt/modules/win_useradd.py

```diff
@@ -194,7 +229,7 @@ def update(name,
            expiration_date=None,
            expired=None,
            account_disabled=None,
-           unlock_account=None,
+           lock_account=None,
```

Comment:
While this is cleaner, this breaks backwards compatibilty. Should this go through a deprecation path, or nobody is worried about this. @dwoz ?

Reply:
@rares-pop I agree that it's unlikely many people use this functionality. However, I do think we need to go through the deprecation path, just to be safe.

Reply:
@syphernl We should follow the depreciation path defined here: https://docs.saltstack.com/en/latest/topics/development/deprecations.html

Reply:
@dwoz The command has been re-added and marked as deprecated.